packages/d/datatables-plugins.json

```diff
@@ -6,6 +6,7 @@
     "jQuery",
     "plugins"
   ],
+  "filename": "media/js/dataTables.dataTables.min.js",
```

Comment:
```suggestion
  "filename": "sorting/numeric-comma.js",
```
[Let's use a file from the package](https://www.digitalocean.com/community/tools/glob?c=Y29tbWVudHM9dHJ1ZSZnbG9iPSZtYXRjaGVzPWZhbHNlJnRlc3RzPSUyRsUDMEltcG9ydGVkJTIwZnJvbSUyMGRhdGF0YWJsZXMubmV0LXBsdWdpbnMlMjBOUE0lMjBwYWNrYWfISMcOLmpzb27HE2Jvd2VyzBFMaWNlbnNlLnR4dMcSbWFrZS5zaMcOUmVhZG1lLm1kxxBhcGklMkZhdmVyYWdlJTI4JTI5LmpzzR1jb2x1bW7HHHRpdGzdKHPHHW9yZGVy1ilmbkFkZERhdGFBbmREaXNwbGF51SNU5ADazxdDxWVJbmRleFRvVmlzaWJs5AERzybEXVVwZGF01BzGclJvd9kcU3RhcnTSHkZha2VSb3dzcGFu0x1pbHRlckFsbNgbQ2xlYfMA0cYdT25SZXR1ctVYbmRDZWxsUm935QD2ZXPdJE5vZNQiR2V0QWRqYWNlbnT0AVZHZXTmAVnkATnbH%2BUAgtUgSGlkZGVu2n5UZNcX0y9MZW5ndGhDaGFu5QLtzx5NdWx0aeYBNtIdUGFnaW5nSW5mb9Mccm9jZXNzxSBkaWNhdG%2FTQVJlbG9hZEFqYfMA51NldMZ4aW5nRGX1ArZTb3J0TmV1dHJh8wIIU3RhbmRpbmdSZWRyYfMCfucCz1Rv%2BwFt5QNXLm7GY%2FYDX3DlBCR1bXBUb%2BQBz9cl6QEN1iByb3fHGXNoyQvSJOgD7mdlbmVyYXT3BBpzdW3QGWJ1dHRv5AQtRsYKLmRvd27kAXvKI%2BQFHFJlbuUER0bkA8x0aW3rAhDNH2VsbGlwc2nrAk3NH2ludOsBkM0bcGVy5AMWYWdlQmFyy0BmZWF0dXJl5ACiYWxwaGFiZXRTZWFyY2jmAI1hVOYFts4cLmNz30DfQN9%2F3z8ubWnrBELLQ2NvbmRpdGlvbmFs5gMizkbRH9VFZGVlcExpbmvOPMgW3zPVM%2FgAr2zlBAHEI8VFyjrLGfYBa986xjrfc9o5%2BACwbWFy5AEAxznnB9DFFfUApdovZXM232HUMv8Al9M23y7PLsRg%2FwDJ1DPUZeQEnFJlc2l6Ze4BZsoY3zfZN99yxjtp5QVTaHRtbN8pc2Nyb2xs5AMK1i1yb3dGaWxs7gCOxxX1AIfdMfgAvMo1%2FAC55gCs9AEfzBrVYd87yDv4AKDQP3luYW1pY%2F8Ap84tVmjfL%2F8BusorVG9Ub3D0AMXFGfsAwN85%2BQD95QVzRmFk8AE6yRj2AzXfOMQ432%2FYN%2FkDqN88yDzad0hpZ2hsaWdodNRAyR3cfd9CyEL%2FAIPfQWh0%2FwDIUGFu5AEDY3VzdG9tT3DkBvNz%2BAJVzDHRccQY%2FADv3Tj%2FAKDYN%2F8B3988%2FwEX2jtz33T%2FA53NKXZlcnRpY2Fs2CxsaeQLIENoaWxk7gZVzBr2AMnSO%2BgAy8w77gEI6AvN5gVrLWJhc%2BQPnEZy5Ax5X%2BQKcuwILN0tbnVtYmXtCj%2FXL%2BUAlVRvb2xzLlNob3dTZWxlY3RlZE9ubOsMTsw9dHlwZck%2BYWPkCqwt5wv%2FaXPsAkzYNGRpYWNyaXRpY3PfOM845AGj3yfEJ3Bob25lTuUA%2F8ouaTE4biUyRkFmcmlrYWFucy5sYW5nzxxsYmFuaWFu1BttaGFyaWPUGnJhYtcZbWXYTnplcmJhaWrVHUJhbmdsYdUZc3F1ZdQZZWxhcnVz1mxCdWxnYXLWHENhdGFs1hpoaW5lc2UtdHJh6AuH2ibUGnJvYXTXdXplY2jTGERhbmlz1Rl1dNUxRW5nbNYyRXNwZXJhbnRv1Rx0b%2FcBl0ZpbGlwaW7UNkZpbvcAnkZyZW71AJ9HYWxpY9ZpR2VvcmfYG3Jt1hlyZWVr1Bh1amFyYXRp0xtIZWJyZXfUGWluZNUxdW75AixJY2Vs5BAs9ALrSW5kb25l5AKBLUHkA5RuYXRpdvQCF8op1B1y9gFiSXRhbNYySmFwYfcCgUthemFr1E5LaG1lctQYb3Jl1WVLdXJk9gCZS3lyZ3l60xlMYfQCRExhdHb2AMhMaXRodfgEe01hY2Vk%2BAKzTWHkEfTTGG9uZ2%2F3ATZOZXBhbPQB%2F05vcndl5AKYLUJva231A%2FLKI055bm9yc%2FQCklBhc2j1A3tQZXL3AftQb%2FcDylBvcnR1Z3XkBItCcmFzafQAlMok0x1Sb234AVlSdXP3AI9TZXJiaWFuX2xhdGnbINQaaW5oYfUFvlNsb3Zh9AEvxBn4BilTcPgFFVN3YWhp9QHGU3dl9wKwVGFqafQAglRhbfUBWXRlbHVnddQxaGHUelR1cmvWelVrcmFp9wDkVXJk1GRVemL1BPBWaWV0bmFt9gHuV2Vs7wCAbnRlZ%2BQFH2%2FEH2Jvb3RzdHJh5A1bMe4Jk8kb6wrQ3z3UPeoIb9k8Mt953z3fed881TztCoLeQPELV9ovM9hv%2FwDo3j3%2FAKjdPP8A6MxA%2FwDo0C9SRUFETUXrGuLaL2ltYWfmDB9vcnRfYXNjX2Rpc%2BQAqmQucP8Ct9U%2F3zbUNmJvdGjfN9Q3ZGX%2FAK3fQMRA2Tdmb250LWF3ZXNvbeQNlslG%2BgGEzzLrAfLEGkHGGfkCbdM%2B5w3Y33HJM2ZvdW5kxRzcM8gk8ACiyTz6AKHfO%2FkCzN86xDr7AwrNPv8Crd9A%2FwKu3zf%2FAq%2FfOP8CsN9B%2FwKx0jj%2BBC%2FNLP8ELGpxdWVyeXXkGqTrAbzIFvkCL9839wHt3zb7AiffOusRotk4%2FwE1yyrxATNw5BoTyCnmG2nsERnOH3h0atgcZm91cl%2FnG%2B7XInVsbOgRS19ub1%2FfbfEA1M8ecHXrFFHNHGrlGufkHgLvAVPJGNc56hgY1yDlEdDYHWltcGxlX2luY3Jl5CI5YWxf8wds1DP9ARTkAz%2FmEZkuRFNfU3RvcugiMcoaYWJzb2x17CEPyxxsdC1zdOQR3NUebnRpLXRo1jpueS3mAJvUHmJyYWNrZXTkEndn5g6k0yVDYXNlLVNlbnNp2CJjaGFw7R%2FszBvmETn6AL5jdXLkEDPrE1zMHOQWBi3kAf8tc291cmPlBpxvbS1jaGVja2Jv6x983zXENfAB%2Bt8zxDN0ZXjWMeQRtfsA1eQUmi1kZC1NTU0teXl59QDdxiT1AVHFG2V12xty6yE%2Bzx116xw9zhvkH54tbW%2FkAtndI3X1ArvrFKbEFdQjZW51bdQYZmFyc2nnAojVXGZpbGUt7hsEyx1vcm1hdHRlZNxC7iBAyxhwLWFkZHJlc9U2bW9udGhZ7SP2yh1u5Ba7YWwt5QEUZGVs7CN40SbVG%2BUQWvwAtm51bS3uFcbNHGVyaWMt5CbE1np1bVP5AoLnITXXG%2BUP6dMbc2NpZW50aWZpY9UeaWdu5gGG1R7lAIxN%2FAFS5QFKZWxhcHNlZC1kaG31ARTuIkTMGHRsZcR95AEW2iH6A5V05g%2BQ0SLlFy1kZXRlY%2BcF0%2FIEk9Ej8QNq0SLzAtPSJOwC2tso9ALF0SXyAkzUI%2B0CUw%3D%3D)